client/src/cli/apiToken.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  AccessTokenError,
+  getRmmbrAccessTokenPath,
+} from "./rmmbrAccessToken.ts";
+import { err, ok, Result } from "./deps.ts";
+
+const rmmbrServer = Deno.env.get("RMMBR_SERVER");
+
+export const apiToken = async () => {
+  const result: Result<string, AccessTokenError> = await (
+    await getAccessToken()
+  ).match({
+    Err: async (e) => await err(e),
```

Comment:
would `Err: err` just work?
actually why do we need this line, doesn't it automatically propagate it?

at any case you can drop the `await`

why it attracts my attention: becasue it's like an line of code without information

Reply:
Deno yells at me when I declare an async function without an await.. I'll wrap in a Promise, this is a really ugly hack

Couldn't figure out what is the correct way to use true-myth to chain async functions. =(